### PR TITLE
integration tests: post-mortem disk usage and pod describe

### PR DESCRIPTION
**Pull request by tstromberg**
For assistance in debugging DiskPressure and CrashLoopBackoff issues.

### CODE REVIEW

File: test/integration/helpers.go [hunk 1]
```diff
@@ -204,33 +204,52 @@ func clusterLogs(t *testing.T, profile string) {
...
 	t.Logf("-------------------post-mortem minikube logs----------------------")
+
 	rr, err := Run(t, exec.Command(Target(), "-p", profile, "logs", "--problems"))
 	if err != nil {
 		t.Logf("failed logs error: %v", err)
 		return
 	}
 	t.Logf("%s logs: %s", t.Name(), rr.Output())
 
+	t.Logf("------------------post-mortem disk usage-------------------")
```
Comment (medyagh) [inline test/integration/helpers.go:215]: please use the same amount of dashes

Comment (medyagh) [reply test/integration/helpers.go:215]: the line length should be same as all over line lengthes

**Comment by medyagh** [comment]
containerd test failure : 
https://github.com/kubernetes/minikube/issues/6956

**Comment by tstromberg** [comment]
Counting dashes seems ridiculous :(

**Comment by medyagh** [comment]
> Counting dashes seems ridiculous :(

no need to count I didnt count it, just look at the where the line ends for previous one and fill it up till it is same

File: test/integration/helpers.go [hunk 2]
```diff
@@ -203,35 +203,52 @@ func clusterLogs(t *testing.T, profile string) {
...
+	t.Logf("------++> post-mortem[%s]: describe node", t.Name())
 	rr, err = Run(t, exec.Command("kubectl", "--context", profile, "describe", "node"))
 	if err != nil {
 		t.Logf("%s: %v", rr.Command(), err)
 	} else {
 		t.Logf("(dbg) %s:\n%s", rr.Command(), rr.Output())
 	}
-	t.Logf("------------------------------------------------------------------")
+
+	t.Logf("------++> post-mortem[%s]: describe pods", t.Name())
```
Comment (medyagh) [inline test/integration/helpers.go:244]: this is describes all of them how about we desribe all the Not-running ones?

Comment (medyagh) [reply test/integration/helpers.go:244]: also conisder adding 'kubectl cluster-info'

Comment (medyagh) [reply test/integration/helpers.go:244]: here is the command
```kubectl get pods -A --field-selector=status.phase!=Running```

File: test/integration/helpers.go [hunk 3]
```diff
@@ -203,35 +203,52 @@ func clusterLogs(t *testing.T, profile string) {
...
 	rr, err = Run(t, exec.Command("kubectl", "--context", profile, "describe", "node"))
 	if err != nil {
 		t.Logf("%s: %v", rr.Command(), err)
 	} else {
 		t.Logf("(dbg) %s:\n%s", rr.Command(), rr.Output())
 	}
-	t.Logf("------------------------------------------------------------------")
+
+	t.Logf("------++> post-mortem[%s]: !running pods", t.Name())
+	rr, err = Run(t, exec.Command("kubectl", "--context", profile, "describe", "po", "-A", "--field-selector=status.phase!=Running"))
```
Comment (medyagh) [inline test/integration/helpers.go:245]: I think the --field-selector works only on get pods, and then once u have the list, u can iterate through them.

Comment (medyagh) [reply test/integration/helpers.go:245]:
```
 $ kc get po -A --field-selector=status.phase=Running
NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
kube-system   coredns-66bff467f8-cpzcv           0/1     Running   0          20s
kube-system   coredns-66bff467f8-lnb2m           1/1     Running   0          20s
kube-system   etcd-minikube                      1/1     Running   0          35s
kube-system   kindnet-wrpm7                      1/1     Running   0          20s
kube-system   kube-apiserver-minikube            1/1     Running   0          35s
kube-system   kube-controller-manager-minikube   1/1     Running   0          35s
kube-system   kube-proxy-cxfqr                   1/1     Running   0          20s
kube-system   kube-scheduler-minikube            1/1     Running   0          35s
kube-system   storage-provisioner                1/1     Running   0          32s
medmac@~/workspace/minikube (kic_stop_improve) $ kc describe po -A --field-selector=status.phase=Running
Error: unknown flag: --field-selector
See 'kubectl describe --help' for usage.
```

Comment (tstromberg) [reply test/integration/helpers.go]: Bummer. Will leave the iteration for a subsequent PR.

**Pull request merged**